src/VisualStudio/CSharp/Impl/CodeModel/CSharpCodeModelService.cs

```diff
@@ -3021,6 +3021,14 @@ protected override ITypeSymbol GetTypeSymbolFromPartialName(string partialName,
 
         public override ITypeSymbol GetTypeSymbolFromFullName(string fullName, Compilation compilation)
         {
+            // Some clients pass us an assembly qualified name, which we don't handle.  Just lop off the
+            // assembly info if so.
+            var index = fullName.IndexOf(',');
```

Comment:
Well, this won't actually be sufficient.  Commas can of course appear in language specific type names in array rank specifiers, generic specifiers, etc.